### PR TITLE
Updating broadway and elsa to latest

### PR DESCRIPTION
**Pull request by jeffgrunewald**
Updating broadway to latest 0.5.0 api and updating elsa to latest 0.11.1 to address bug in consumer group acknowledgement.

### CODE REVIEW

File: lib/off_broadway/kafka/showtime_handler.ex [hunk 1]
```diff
@@ -25,22 +25,21 @@ defmodule OffBroadway.Kafka.ShowtimeHandler do
   def init(args) do
     broadway_module = Keyword.fetch!(args, :broadway_module)
     opts = Keyword.get(args, :opts, [])
+    producer_selector = Keyword.get(args, :producer_selector, &default_selector/1)
```
Comment (bbalser) [inline lib/off_broadway/kafka/showtime_handler.ex]: I am not sure I understand the use case where a user would want to control which producer is selected.

Comment (jeffgrunewald) [reply lib/off_broadway/kafka/showtime_handler.ex]: I can’t think of one personally either beyond arbitrarily selecting the first one or a random one didn’t feel quite right as the only available option. I’m not married to it though.

Comment (bbalser) [reply lib/off_broadway/kafka/showtime_handler.ex]: it doesn't hurt anything, my only concern is adding confusion for a user when reading the documentation.  Other than that, I am good.

Comment (jeffgrunewald) [reply lib/off_broadway/kafka/showtime_handler.ex]: agreed; i don't want to include unnecessary confusion or complexity. it's definitely cleaner just making the default function the only supported behavior.

**Pull request merged**